README.md

```diff
@@ -32,12 +32,12 @@ Enable 'Bookmark' in Administration » Islandora » Solr index » Solr Settings
 
 Set the permissions in Administration » People (admin/people/permissions). There are permissions to administer, use, and share bookmarks. Anonymous users can never share bookmarks.
 
+The bookmark tool only displays on Islandora object pages for logged-in Drupal users with existing bookmark lists. The admin page has an option to create a default bookmark list for each individual user. Users cannot delete their default lists. The first time you enable default bookmark lists, you will have to log out and log back in before you can view your default list.
```

Comment:
Thanks for clarifying, Jordan. Would this be more accurate for the first sentence?

"The bookmark tool only displays on Islandora object pages if the user has permission to add bookmarks to an existing list."